infra/recipes/docker-compose/oh-abs-spark/cluster.yaml

```diff
@@ -0,0 +1,37 @@
+cluster:
+  name: "LocalFSCluster"
+  storage:
+    type: "hadoop"
+    uri: "hdfs://namenode:9000/"
+    root-path: "/data/openhouse"
+  storages:
+    default-type: "adls"
+    types:
+      adls:
+        rootpath: <container name>@<storage account name>.dfs.core.windows.net 
+        endpoint: "abfs://" # populate your values here
+        parameters:
+          adls.auth.shared-key.account.name: # add your Azure Storage Account name here
```

Comment:
ok to check in admin/password for docker setup

Reply:
This username/password is specific to the user's Azure storage account so there are no single values I can add here.

Reply:
This name/key corresponds to the user's Azure storage account, so I can't prepopulate it here if that is what you are referring to